canonical/charmhub.io
# chore: Clean up beta store

## Done
Removed beta store code from app and integrated it into the main application

## How to QA
- Go to https://charmhub-io-1959.demos.haus
- Try the search, filters, pagination and confirm it works as it does in production
- Go to https://charmhub-io-1959.demos.haus/topics
- Try the search, filters, pagination and confirm it works as it does in production
- Go to a charm page e.g. https://charmhub-io-1959.demos.haus/data-platform-libs
- Click around and confirm it works as it does in production
- Go to a bundle page e.g. https://charmhub-io-1959.demos.haus/canonical-livepatch-onprem
- Click around and confirm it works as it does in production

## Testing
- [x] No testing required (explain why): Only structural changes

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-13869
